.helm.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+function download {
+	curl https://storage.googleapis.com/kubernetes-helm/helm-v2.5.1-linux-amd64.tar.gz --output /tmp/helm.tar.gz --silent
+
+	tar -x -C /tmp -f /tmp/helm.tar.gz linux-amd64/helm
+
+	if [[ $TRAVIS_TAG ]]; then
```

Comment:
Maybe, but it was working in the two case. Install `aswcli` only if is a tag push